Signal/src/util/DeviceSleepManager.swift

```diff
@@ -71,7 +71,7 @@ import Foundation
 
         if UIApplication.shared.isIdleTimerDisabled != shouldBlock {
             if shouldBlock {
-                Logger.info("\(self.TAG) \(#function): Blocking sleep")
```

Comment:
> there could be multiple block objects.

I've added a little clearer logging message in this case: `Blocking sleep because of: FOO and 2 others`

Though in practice it's unlikely to hit this. We only log when sleep-blocking changes - so adding a second blocking object will not log unless both of those objects were added at the same time. 

I've also added some asserts to make sure we're only interacting with the `blocks` state on the main thread.